### ISSUE TITLE
W3C /status endpoint breaks JSONWP backwards compatibility

### ISSUE THREAD

**Issue by dpgraham** [appium/appium#13161]
## The problem

`/status` is returning W3C by default which will break backward compatibility with any clients or scripts that expect `/status` to return JSONWP.

Previous Appium versions supported both, by having a combined JSON output that was JSONWP _and_ W3C compatible. This seems to have been removed.

See the "Steps to Reproduce" to get a better idea

## Environment

* Appium version (or git revision) that exhibits the issue: 1.15
* Last Appium version that did not exhibit the issue (if applicable): v1.14.0
* Appium CLI or Appium.app|exe: CLI

## Link to Appium logs

Code to Reproduce provides output

### v1.14.0 (working)

```
git checkout v1.14.0
rm -rf node_modules && npm install
node .
```

Then open `localhost:4723/wd/hub/status` and the response is

```json
{
    "status": 0,
    "value": {
        "build": {
            "version": "1.14.0",
            "git-sha": "9c325d3d355837e49d352f78357761571644ce4f",
            "built": "2019-07-04 18:41:25 +0200"
        }
    },
    "sessionId": null
}
```

### 1.15 branch (not working)

```
git checkout -b 1.15
git pull origin 1.15
rm -rf node_modules && npm install
node .
```

Then open `localhost:4723/wd/hub/status` and the response is

```json
{
    "value": {
        "build": {
            "version": "1.15.0-beta.0",
            "git-sha": "15d3774b013e736b163902f9c67a417c8aa3ebf5",
            "built": "2019-08-28 09:53:31 -0400"
        }
    },
    "sessionId": null
}
```

The status key is missing, and this will break any test that expects it to be there. 

I discovered this because it broke one of our test pipelines in Sauce Labs.

**Comment by mykola-mokhnach** [comment]
No, previously it was just a coincidence since we didn't have any downstream driver running in W3C mode. According to W3C standard `sessionId` field could be removed from the response body as well.

**Comment by dpgraham** [comment]
`/status` needs to support W3C and JSONWP. Since `/status` doesn't have a session associated with it, it can't assume that the response is W3C.

**Comment by dpgraham** [comment]
I'll look at what Chromedriver does.

**Comment by dpgraham** [comment]
Alright so Chromedriver does it like this:

```
{
    "value": {
        "build": {
            "version": "76.0.3809.126 (d80a294506b4c9d18015e755cee48f953ddc3f2f-refs/branch-heads/3809@{#1024})"
        },
        "message": "ChromeDriver ready for new sessions.",
        "os": {
            "arch": "x86_64",
            "name": "Mac OS X",
            "version": "10.14.6"
        },
        "ready": true
    }
}
```

**Comment by dpgraham** [comment]
@jlipps @imurchie https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#responses

^ does including `status` in the response object apply to the `/status` endpoint?

**Comment by mykola-mokhnach** [comment]
Would https://github.com/appium/appium-base-driver/pull/353 be ok?

**Comment by dpgraham** [comment]
Yeah that looks good @mykola-mokhnach .... better to be safe, can't hurt to have an extra `status` and `sessionId` key.

**Comment by mykola-mokhnach** [comment]
The fix has been published to appium@beta. Please confirm the issue is resolved

**Comment by dpgraham** [comment]
Sorry, let me check again.

**Comment by dpgraham** [comment]
It's showing `sessionId` but not `status`...

Repro steps:

```
# Checkout latest master
rm -rf node_modules && npm install
node .
```

Visit `localhost:4723/wd/hub/status`

The output is:

```json
{
"value": {
"build": {
"version": "1.15.0-beta.0",
"git-sha": "d13dff99a3e6ed34886fd4b7d6015a4f32ccafa8",
"built": "2019-09-04 09:00:12 +0900"
}
},
"sessionId": null
}
```

**Comment by mykola-mokhnach** [comment]
Fixed in base driver 4.0.2
